### PR TITLE
Implemented ReportsPage components 

### PR DESCRIPTION
**Pull request by Mihaidaniel22**

### CODE REVIEW

File: src/components/homepage/DateNavigation.jsx [hunk 1]
```diff
@@ -72,7 +75,11 @@ export const DateNavigation = ({ selectedDate, onDateChange }) => {
           }}
         >
           <LocalizationProvider dateAdapter={AdapterDateFns}>
-            <DateCalendar views={["day", "month", "year"]} />
+            <DateCalendar
+              views={["day", "month", "year"]}
```
Comment (en-teodora-velicu) [inline src/components/homepage/DateNavigation.jsx:79]: this array can be extracted as a const outside of the function.

File: src/components/reports/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Box } from "@mui/material";
+import React from "react";
+import Typography from "@mui/material/Typography";
+import "./Categories.css";
+
+const Categories = () => {
+  return (
+    <Box className="categories-container">
+      <Box className="category-box">
+        <Typography className="category-title">Food & Restaurants</Typography>
```
Comment (en-teodora-velicu) [inline src/components/reports/Categories.jsx:10]: This is duplicated code. A box can be set as a reusable component SimpleCard (or sth like that) and you will send title and total as params

File: src/components/reports/ReportsNavigation.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          open={open}
+          anchorEl={anchorEl}
+          onClose={handleClose}
+          anchorOrigin={{
+            vertical: "bottom",
+            horizontal: "left",
+          }}
+        >
+          <LocalizationProvider dateAdapter={AdapterDateFns}>
+            <DateCalendar
```
Comment (en-teodora-velicu) [inline src/components/reports/ReportsNavigation.jsx:78]: The calendar should be a reusable component. Try and optimize it to fit our needs

File: src/components/reports/ReportsPieChart.jsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import { Typography } from "@mui/material";
+import React from "react";
```
Comment (en-teodora-velicu) [inline src/components/reports/ReportsPieChart.jsx:2]: We should have a reusable component for pie charts. Please update for both homepage and reports. Try to adapt the props to fit our needs

File: src/components/reports/ReportsPieChart.jsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+const data = [
+  { label: "Food & Restaurants", value: 400, color: "#4F46E5" },
+  { label: "Car", value: 300, color: "#1EB3DB" },
+  { label: "Subscriptions", value: 300, color: "#E83FAB" },
+  { label: "Entertainment", value: 200, color: " #FF8084" },
+  { label: "Coffee", value: 278, color: "#9FD11F" },
+];
+
+const ReportsPieChart = () => {
```
Comment (en-teodora-velicu) [inline src/components/reports/ReportsPieChart.jsx:15]: Same thing here. Reports should be a reusable comp

**Review by en-teodora-velicu — CHANGES_REQUESTED** [review]

**Review by en-teodora-velicu — APPROVED** [review]

**Review by CristiStroescu — APPROVED** [review]

**Pull request merged**